### PR TITLE
Feature/all habits gallery view

### PR DESCRIPTION
**Pull request by HryshkoAnton**
Add All Habits page gallery view

### CODE REVIEW

**Comment by mplesha** [comment]
conflicts

**Review by sofiiamatsyhin — APPROVED** [review]

**Review by yulkats — APPROVED** [review]

**Comment by mplesha** [comment]
conflicts

File: src/app/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { LocalStorageService } from './../../../../../service/localstorage/local-storage.service';
 import { Component, OnInit } from '@angular/core';
 import { HabitTab } from '../../../models/habit.model';
 import { HabitItemModel } from '../../../models/habit-item.model';
```
Comment (mplesha) [inline src/app/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts:4]: apply aliases of paths

File: src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+div {
```
Comment (mplesha) [inline src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.scss:1]: why element `div` but not some class?

File: src/app/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts (hunk 1 above)
Comment (HryshkoAnton) [reply src/app/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts]: done

File: src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.scss (hunk 2 above)
Comment (HryshkoAnton) [reply src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.scss]: it was just mock styles, this styles will be part of work of another developer. I removed it.

File: src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<div class="card"></div>
```
Comment (mplesha) [inline src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.html:1]: fix

Comment (HryshkoAnton) [reply src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.html]: fixed

**Pull request merged**